Recipes/JsonConfiguration/JsonConfiguration.cs

```diff
@@ -63,7 +62,7 @@ public static JsonSerializerSettings CompactSerializerSettings
                 {
                     Formatting = Formatting.None,
                     NullValueHandling = NullValueHandling.Ignore,
-                    ContractResolver = new CamelCasePropertyNamesContractResolver(),
+                    ContractResolver = CamelCasePropertyNamesWithConstructorParametersRequiredContractResolver.Instance,
```

Comment:
we've discussed - going to implement a new settings called MinimalSerializerSettings